float64 x
float64 y
float64 z
int64 type
int64 source
---
float64 x1
float64 t1
float64 y1
float64 z1
int64 sense
int64 type1
int64 serverCCReply
int64 recovery
float64 state1
float64 state2
float64 state3
float64 n